torch_np/_ndarray.py

```diff
@@ -358,8 +357,7 @@ def reshape(self, *shape, order="C"):
 
     def sort(self, axis=-1, kind=None, order=None):
         # ndarray.sort works in-place
```

Comment:
If so, do we need to do a `copy_`?

Reply:
We have to perform the copy in-place to preserve the semantics:
```python
>>> import numpy as np
>>> a = np.array([1, 3, 2])
>>> id(a)
140581065530512
>>> a.data
<memory at 0x7fdb94820dc0>
>>> a.sort()
>>> a
array([1, 2, 3])
>>> id(a)
140581065530512
>>> a.data
<memory at 0x7fdb94820dc0>
```

Reply:
Indeed. `self.tensor.copy_(_funcs._sort)` it is then.